community/concepts/cred.md

```diff
@@ -0,0 +1,20 @@
+# Cred
+
+SourceCred, as the name might suggest, is all about attributing cred. Cred is a metric that describes every contribution and contributor in a project to give a sense of how important they were.
+
+For example, forum posts can earn cred. If I post on the [sourcecred discourse](https://discourse.sourcecred.io/) and earn 5 cred, and another user posts and their post earns 10 cred, we can say that the other post is considered twice as important as my post.
+
+The contributions are arranged in a [graph](https://en.wikipedia.org/wiki/Graph_(discrete_mathematics)) where contributions are nodes, and have edges indicating how they relate to other contributions. For example, if someone writes a reply to my post, there will be an edge connecting the reply and my post. Edges have "types" that give some information about what kind of edge they are; that edge might have an "IS_REPLY" type. The connections are also directional, meaning their are relationships that connect nodes to it, and relationships that connect nodes from it. An edge without a direction is considered bi-directional.
+
+Contributors — like you or me — are also nodes in the graph, and are connected to the contributions that they create. So there is an "AUTHORS" edge connecting me (the author) to my post.
+
```

Comment:
This is a good sentence, but feels awkward at the end of the para. Suggest moving it to after the first sentence introducing PageRank.

Reply:
Do you mean after the contributions are arranged in a graph sentence? I just reworked this paragraph, and let me know when I push what you think.